mlflow/server/js/src/experiment-tracking/components/RunView.js

```diff
@@ -125,6 +125,26 @@ export class RunViewImpl extends Component {
     const sourceVersion = Utils.getSourceVersion(tags);
     const entryPointName = Utils.getEntryPointName(tags);
     const backend = Utils.getBackend(tags);
+
+    if (Utils.getSourceType(tags) === 'PIPELINE') {
+      const profileName = Utils.getPipelineProfileName(tags);
+      const stepName = Utils.getPipelineStepName(tags);
+      runCommand = '';
+      if (sourceName) {
+        runCommand += `git clone ${sourceName}\n`;
+      }
```

Comment:
I am assuming the sourceName and the sourceVersion is automatically tracked while to create a run?

Reply:
I assume it's always here, at least it something that deducted from the code.